### PR TITLE
Remove experimental microservices from service-all.

### PR DESCRIPTION
**Pull request by dacolgit**
Removed the following services from services-all container:
* Telemetry CDM Processor
* OPC Gateway
* Http Tunnel Processor

### CODE REVIEW

File: Industrial-IoT.sln [hunk 1]
```diff
@@ -25,8 +25,6 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.IIoT.Servic
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.IIoT.Services.OpcUa.Registry.Sync", "services\src\Microsoft.Azure.IIoT.Services.OpcUa.Registry.Sync\src\Microsoft.Azure.IIoT.Services.OpcUa.Registry.Sync.csproj", "{98CC86E7-6BD0-4B5A-B71B-C8F57C98E081}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.IIoT.Services.OpcUa.Twin.Gateway", "services\src\Microsoft.Azure.IIoT.Services.OpcUa.Twin.Gateway\src\Microsoft.Azure.IIoT.Services.OpcUa.Twin.Gateway.csproj", "{A8C4FFE0-20D8-40F4-8B9F-26FE9D013E2F}"
```
Comment (cristipogacean) [inline Industrial-IoT.sln]: let's keep the projects for now in the solution. We are still generating images for these services, so I prefer to keep the projects in solution until we completely remove the projects from the repository.

Comment (dacolgit) [reply Industrial-IoT.sln]: added

Comment (cristipogacean) [reply Industrial-IoT.sln]: let's revert the other 2 projects from the solution as well. better revert completely the change to .sln

Comment (dacolgit) [reply Industrial-IoT.sln]: reverted all changes in solution file

**Review by cristipogacean — APPROVED** [review]

**Review by karok2m — APPROVED** [review]

**Pull request merged**